kotest-property/kotest-property-core/src/jvmTest/kotlin/io/kotest/property/core/DelegateTest.kt

```diff
@@ -0,0 +1,97 @@
+package io.kotest.property.core
+
+import io.kotest.core.spec.style.FunSpec
+import io.kotest.matchers.shouldBe
+import io.kotest.matchers.string.shouldEndWith
+import io.kotest.matchers.string.shouldHaveLength
+import io.kotest.matchers.string.shouldStartWith
+import io.kotest.property.Arb
+import io.kotest.property.ShrinkingMode
+import io.kotest.property.arbitrary.int
+import io.kotest.property.arbitrary.string
+import kotlin.time.Duration.Companion.milliseconds
+
+class DelegateTest : FunSpec() {
+   init {
+      proptest("delegate syntax 1") {
+
+         iterations = 42 // run this property test 42 times
+         failOnSeed = true // will fail if a seed is set
+         edgecaseFactor = 0.1 // 10% of generated values will be edgecases
+         shouldPrintGeneratedValues = true // output each generated value
```

Comment:
I like to have a configuration inside the test itself.

It goes a bit against what we usually do with `.config(xx)`

Reply:
Yes, its different. I think more in line with current kotlin idioms though ?

Reply:
this is really convenient. is there a plan to provide a function, something like 'printCurrentConfig()` so that users can discover what settings are available, and which settings are in effect in this context?

Reply:
Good idea